websites/C/Craiyon/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+	clientId: "1015402986534608948",
+});
+
+let elapsedTimestamp: number = null,
```

Comment:
```suggestion
let browsingTimestamp: number = null,
```

Reply:
You should update the change at all references too